content/installation/netcore/install/NetCoreInstallation.md

```diff
@@ -5,56 +5,38 @@ description: "Contrast .NET Core agent installation instructions"
 tags: "installation agent .NET Core windows linux"
 -->
 
-## The Basics
+## Install Overview
 
-To install the .NET agent, complete the following steps. The installation process for self-hosted applications also applies to IIS Express users.
+This document details the basic install of the .NET Core agent. The installation process for self-hosted applications also applies to IIS Express users.
```

Comment:
Can you explain the change? The deleted text is actually better style than the change. Is it incorrect to say:

To install the .Net Core agent, complete the following steps. Note: the installation process for self-hosted applications also applies to IIS Express.

Reply:
Yea, good point.  Fixed as you requested.